src/zFooter/Footer.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import styles from "./Footer.module.scss";
+
+export function Footer() {
+  return (
+    <div className={styles.redRibbon}>
+      <span className={styles.footerText}>
+        <Link className={styles.link} to="/impressum">
```

Comment:
I'd advise against using magic strings. While it is common in the FE world to see this, it can create myriad problems. Instead, you can use an JS object (also called a `pojo`) that has your string constants in it, for example:

```javascript
export const appPaths = {
  impressum: "impressum",
  contact: "kontakt",
  // ...
}
```

For more information, see: https://softwareengineering.stackexchange.com/questions/365339/what-is-wrong-with-magic-strings

Reply:
Thanks for that 👍

Reply:
Should I create the object in a separate folder or in the same folder as the footer?

Reply:
That works for now 😄

Reply:
💯